pages/linux/fdisk.md

```diff
@@ -0,0 +1,19 @@
+# fdisk
+
+> Manipulate disk partition table.
+
+- List  the  partition  tables for the sda disk and then exit:
+
+`fdisk -l /dev/sda`
+
+- Print the size (in blocks) of /dev/sda1:
+
+`fdisk -s /dev/sda1`
+
+- Create/remove/change partitions in the device:
```

Comment:
Ah, yeah - I see. it's this command that remains as a sort of interactive shell. In that case, perhaps we could alter this description to reflect that rather than saying "and then exit" above?